docs/administration/backup-restore/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Backup And Restore Harbor With Velero  
+weight: 50
+---
+
+Backup and restore is important for the disaster recovery and data migration. With [Velero](https://velero.io/) the administrator can back up and restore Harbor instances without completed shutdown.  
+
+Please note that:
+* This section only covers the Harbor instances deployed in Kubernetes cluster by Harbor helm chart
+* For simplicity, we only back up a subset of Harbor's resources and data:
+  * All Harbor related Kubernetes resources(deployments, statefulsets, services, configmaps, etc.)
+  * Data in the PersistentVolumes of internal database, registry, chartmuseum, jobservice and Trivy. We don't back up the data of Redis, see the [Limitations](#limitations) part for more details of the impact
+* The backup is crash consistent, not application consistent, some data will lose after restore, see the [Limitations](#limitations) part for more information
+
+## Install Velero
+Install Velero CLI and server according to the [official documentation](https://velero.io/docs/v1.9/basic-install/).
+
+**Note**: Depending on the data size, you may need to increase the CPU or memory resources available to Velero, especially if you are using Restic. Refer to the [doc](https://velero.io/docs/v1.9/customize-installation/#customize-resource-requests-and-limits) for more information.
+
+## Backup Harbor Instance
+### Set Harbor to ReadOnly
+1. Log in to the Harbor portal with an account that has Harbor system administrator privileges.
+1. Expand **Administration**, and select **Configuration**.
+1. Select the **System Settings** tab.
+
+   ![Garbage collection](../../img/set-read-only.png)
+
+1. Select the check box **Repository Read Only** and click the **Save** button to save the configurations.
```

Comment:
for the screent shot, please highlight the Repository Read Only selection.

Reply:
done